src/App.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+import MainPage from './MainPage';
+
+function App() {
+  const [state, setState] = useState({
+    todoInput: '',
+    todos: [],
+  });
+
+  const { todoInput, todos } = state;
+
+  const handleChangeInput = (content) => {
+    setState({
+      todoInput: content,
+      todos: [...todos],
+    });
+  };
+
+  const handleClickAddTodo = () => {
+    setState({
+      todoInput: '',
+      todos: [{
+        id: uuidv4(),
+        content: todoInput,
+      },
+      ...todos],
```

Comment:
```suggestion
      todos: [
        ...todos,
        {
          id: uuidv4(),
          content: todoInput,
        },
      ],
```

spread를 먼저해주시고 새로 추가할 객체를 뒤로 해주세요.

이건 습관적으로 해야됩니다. 새로 추가하거나 기존 속성 값을 덮을 때 먼저 spread를 하고 뒤에 추가합니다.

지금처럼 하시면 원하는 결과가 안 나올 때가 있을 겁니다. 추가된 상태나 객체가 기존 것에 덮이는 경우가 있을 겁니다.

Reply:
코드를 작성할 때 의식에 흐름에 따라 새로운 추가 내용, 그리고 기존 것을 배치했는데 앞으로 주의해야겠습니다!
감사합니다~